app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -242,15 +242,8 @@ def consumer_activate
     #description N_("This service is available for unauthenticated users")
     def server_status
       candlepin_response = Resources::Candlepin::CandlepinPing.ping
-      status = { :managerCapabilities => candlepin_response['managerCapabilities'] + ["combined_reporting"],
-                 :result => candlepin_response['result'],
-                 :rulesSource => candlepin_response['rulesSource'],
-                 :rulesVersion => candlepin_response['rulesVersion'],
-                 :standalone => candlepin_response['standalone'],
-                 :timeUTC => Time.now.getutc,
-                 :version => Katello::VERSION }
-
-      render :json => status
+      candlepin_response[:managerCapabilities] << 'combined_reporting'
```

Comment:
@jturel Is key `managerCapabilities` in symbol or string format? Or both formats are supported?

Reply:
If you inspect the candlepin_response you'll probably find that it's a HashWIthIndifferentAccess: `https://api.rubyonrails.org/classes/ActiveSupport/HashWithIndifferentAccess.html` so it can be used both ways. very handy stuff :)